src/app/components/PortfolioItem/PortfolioItem.tsx

```diff
@@ -97,82 +121,87 @@ class PortfolioItem extends React.Component<Props, {}> {
       numberOfUnits,
       change,
       changePercentage,
-      selectedExchange,
-      supportedExchanges,
       styles,
+      name,
       totalBuyValue,
       totalValue,
-      locked,
-      onClick,
     } = this.props
 
     return (
-      <Box mb={2} className={styles.root} onClick={onClick}>
-        <Box mb={1} className={styles.symbol}>
-          {this.isTempItem() ?
-            <Input
-              blurOnInput
-              placeholder={'e.g. eth'}
-              defaultValue={symbol}
-              handleReturn={(_e, val) => this.handleSymbolChange(val)}
-            /> :
-            <span>{symbol}</span>
-          }
-        </Box>
-          <Box mb={1}>
-          {this.isCoinSelected() &&
-            <select value={selectedExchange || 'default'} onChange={this.handleExchangeChange}>
-              <option value={'default'}>Default</option>
-              {supportedExchanges!.map(item => <option key={item} value={item}>{item}</option>)}
-            </select>
-          }
-          </Box>
-        <Box mb={1}>
-          <span>Buy Price: </span>
-          <Input
-            blurOnInput
-            disabled={locked}
-            defaultValue={buyPrice.toString()}
-            handleReturn={(_e, val) => this.handleBuyPriceInput(val)}
-          />
-        </Box>
-        {!locked &&
-          <Box>
-            <Box mb={1}>
-              <span>Buy amount: </span>
-              <Input
-                blurOnInput
-                disabled={locked}
-                handleReturn={(_e, val) => this.handleAmountInput(val)}
-                defaultValue={numberOfUnits.toString()}
-              />
+      <Box mb={2} className={styles.root}>
+        <Flex >
+
+          <Flex w={1/3} style={{alignItems: 'center'}}>
```

Comment:
manrods gali ir be style ta align padaryt. Yra propsas ant to Flex: `align` gal?